plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestingMySqlServer.java

```diff
@@ -30,6 +30,9 @@
 public class TestingMySqlServer
         implements AutoCloseable
 {
+    public static final String DEFAULT_IMAGE = "mysql:8.0.12";
+    public static final String LEGACY_IMAGE = "mysql:5.7.35";
```

Comment:
cc: @grantatspothero 

This should make https://github.com/trinodb/trino/pull/11105 not needed anymore.